.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+For bug reports, please respect the following guidelines and check the boxes accordingly.
```

Comment:
Do we prefer a badly written bug reports to no bug report at all? If so, maybe say so here ("these are suggestions, not rules" or something).